google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/CloudSdkAppEngineHelper.java

```diff
@@ -265,41 +261,6 @@ public void deleteCredentials() {
   }
 
   @NotNull
-  private DeploymentOperationCallback wrapCallbackForUsageTracking(
-      final DeploymentOperationCallback deploymentCallback,
-      AppEngineDeploymentConfiguration deploymentConfiguration,
-      AppEngineEnvironment environment) {
-
-    StringBuilder labelBuilder = new StringBuilder();
-    if (environment == AppEngineEnvironment.APP_ENGINE_STANDARD) {
-      labelBuilder.append("standard");
-    } else {
-      labelBuilder.append("flex");
-
-      if (deploymentConfiguration.isAuto()) {
-        labelBuilder.append(".auto");
-      } else {
-        labelBuilder.append(".custom");
-      }
-    }
-    final String eventLabel = labelBuilder.toString();
-
-    return new DeploymentOperationCallback() {
-      @Override
-      public Deployment succeeded(@NotNull DeploymentRuntime deploymentRuntime) {
-        UsageTrackerProvider.getInstance()
```

Comment:
I think it will be nice to be able to easily compare completion rate of
deployments in our dashboards without having to join them against our
reported exceptions (which have to be manually triggered by the user).


On Wed, Jun 22, 2016 at 11:09 AM, Etan Shaul <notifications@github.com>
wrote:

> In
> google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/CloudSdkAppEngineHelper.java
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/750#discussion_r68071119>
> :
>
> > -      labelBuilder.append("standard");
> > -    } else {
> > -      labelBuilder.append("flex");
> > -
> > -      if (deploymentConfiguration.isAuto()) {
> > -        labelBuilder.append(".auto");
> > -      } else {
> > -        labelBuilder.append(".custom");
> > -      }
> > -    }
> > -    final String eventLabel = labelBuilder.toString();
> > -
> > -    return new DeploymentOperationCallback() {
> > -      @Override
> > -      public Deployment succeeded(@NotNull DeploymentRuntime deploymentRuntime) {
> > -        UsageTrackerProvider.getInstance()
>
> I was thinking about that too. I think cancellation definitely sounds like
> something good for this (since its user initiated). But I'm not totally
> sure myself if success vs failure makes sense? Hard failures will be caught
> and reported to us anyway.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/750/files/5e7f1cae422ad7999dae95775c979bc62363cbcd#r68071119>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AHf5He6ZSy8sRbWC3FtfqhSaGlqnyDXQks5qOVA9gaJpZM4I73oL>
> .
>